pages/services/dcos-monitoring/0.2.1/install/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: layout.pug
+navigationTitle: Install
+title: Install
+menuWeight: 10
+excerpt: Instructions for installing the DC/OS Monitoring service.
+---
+
+# Prerequisites
+
+- DC/OS Enterprise 1.12 or above.
+- [DC/OS CLI](/latest/cli/install/) is installed.
+- You are logged in as a superuser.
+
+## Install package registry
+
+Please follow these [instructions](https://docs.mesosphere.com/1.12/administering-clusters/repo/package-registry/quickstart/) to install the package registry.
+
+# Install DC/OS Monitoring service
+
+## Download the package
+
+<!-- Does this require that the user have a service account? Also, please add a link to the support site. -->
+Download the `.dcos` package of the DC/OS Monitoring Service from Mesosphere support site.
+
+
+## Install the service
+
+<!-- At what point is the "dcos registry add" command added? -->
```

Comment:
@SarahStegall The command is listed [below](https://github.com/mesosphere/dcos-docs-site/pull/1788/files#diff-1d8c85a2425b1692cfd1fe9cc9b1e4a4R33) before the `package install` command - is that sufficient?